examples/flutter_gallery/.idea/runConfigurations/main_dart.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="flutter_gallery" type="FlutterRunConfigurationType" factoryName="Flutter">
+  <configuration default="false" name="main.dart" type="FlutterRunConfigurationType" factoryName="Flutter">
```

Comment:
You mean if you open Idea on `/<flutter root dir>/`? Doesn't that use the runConfig info from https://github.com/flutter/flutter/tree/master/.idea/runConfigurations?

I tested this by opening idea inside each of the examples, e.g.:
```
cd /examples/hello_world/
idea .
```

What folder are you opening?